channeldb/db.go

```diff
@@ -1125,6 +1127,13 @@ func (d *DB) syncVersions(versions []version) error {
 			latestVersion)
 		return ErrDBReversion
 
+	case meta.DbVersionNumber < minUpgradeVersion:
+		log.Errorf("Refusing to upgrade from db_version=%d to "+
+			"lower version=%d. Upgrade via intermediate major "+
```

Comment:
it is not really lower version though? Should be something like "Refusing to upgrade from too old db version=..."?